src/content/app/tools/blast/utils/blastFormValidator.ts

```diff
@@ -0,0 +1,36 @@
+/**
+ * See the NOTICE file distributed with this work for additional information
+ * regarding copyright ownership.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { ParsedInputSequence } from 'src/content/app/tools/blast/types/parsedInputSequence';
+
+export const BLAST_SPECIES_LIMIT = 25;
+export const BLAST_SEQUENCE_LIMIT = 30;
+
+export const isBlastFormValid = (
+  species: string[],
+  sequences: ParsedInputSequence[]
+) => {
+  if (
+    !species.length ||
+    species.length > BLAST_SPECIES_LIMIT ||
+    !sequences.length ||
+    sequences.length > BLAST_SEQUENCE_LIMIT
```

Comment:
return `species.length && species.length <= BLAST_SPECIES_LIMIT && sequences.length && sequences.length <= BLAST_SEQUENCE_LIMIT`?

Reply:
I guess with the current way it is easier to understand what we will be returning?

Reply:
🤷‍♂️ three ORs are about as easy or difficult to understand as three ANDs

Reply:
That's done

I had to check `species.length > 0 && sequences.length > 0` as `1 && true && 0 && false` returns 0